__tests__/intergration.test.js

```diff
@@ -128,6 +129,7 @@ describe("GET: /api/articles/:article_id", () => {
         expect(article).toHaveProperty("created_at");
         expect(article).toHaveProperty("votes");
         expect(article).toHaveProperty("article_img_url");
+        expect(article).toHaveProperty("comment_count");
```

Comment:
When adding new behaviours to previous tasks/endpoints these should be done in new tests to show this as well. Instead of changing/updating previous tests.